modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -206,9 +207,11 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
                 "---------------------------",
                 "",
 
-                { $"aa|autoaccept:|{StandaloneCliConfigKeys.AutoAcceptCerts}:",
-                    "The publisher trusts all servers it is establishing a connection to. WARNING: This setting should never be used in production environments!\n",
+                { $"aa|acceptuntrusted:|{StandaloneCliConfigKeys.AutoAcceptCerts}:",
```

Comment:
that looks like a breaking change to me. I think we should check if the Twin module has the same argument here to have consistency.

Reply:
autoaccept still works, it is just hidden.  Should we merge in twin then we get this for free.